webdriver/tests/bidi/__init__.py

```diff
@@ -71,7 +71,7 @@ def positive_int(actual: Any) -> None:
     assert isinstance(actual, int) and actual > 0
 
 
-async def create_console_api_message(bidi_session, context, text):
+async def create_console_api_message(bidi_session, context: str, text):
```

Comment:
I don't like that we only partly use typing. Can we actually do it all together or not at all?

Reply:
I couldn't figure out what the type of BiDi session is.

Reply:
Fair enough! I missed `text:`. Fixed. Let me know if you find any other missing arguments.